tasks.py

```diff
@@ -22,49 +31,25 @@ def update_file(filename: str, sub_line: Tuple[str, str], strip: str = None):
 
 
 @task
-def setver(_, patch=False, new_ver=""):
-    if (not patch and not new_ver) or (patch and new_ver):
-        raise Exception(
-            "Either use --patch or specify e.g. --new-ver='Major.Minor.Patch(a|b|rc)?[0-9]+'"
-        )
-    if patch:
-        v = [int(x) for x in __version__.split(".")]
-        v[2] += 1
-        new_ver = ".".join(map(str, v))
+def setver(_, ver=""):
+    """Sets the OPTIMADE Python Tools Version"""
 
-    update_file(
-        TOP_DIR.joinpath("optimade/__init__.py"),
-        ("__version__ = .+", f'__version__ = "{new_ver}"'),
-    )
+    match = re.match("v?([0-9]+.[0-9]+.[0-9]+)", ver)
+    if len(match.groups()) != 1:
+        print("Please specify version as 'Major.Minor.Patch' or 'vMajor.Minor.Patch'")
+        sys.exit(1)
+
+    new_ver = match.group(1)
     update_file(
         TOP_DIR.joinpath("setup.py"), ("version=([^,]+),", f'version="{new_ver}",')
     )
 
-    print("Bumped version to {}".format(new_ver))
+    print("Bumped version to {}".format(match.group(1)))
 
 
-@task
-def update_openapijson(c):
-    # pylint: disable=import-outside-toplevel
-    from optimade.server.main import app, update_schema
-    from optimade.server.main_index import (
-        app as app_index,
-        update_schema as update_schema_index,
-    )
-
-    update_schema(app)
-    update_schema_index(app_index)
-
-    c.run(
-        f"cp {TOP_DIR.joinpath('openapi/local_openapi.json')} {TOP_DIR.joinpath('openapi/openapi.json')}"
-    )
-    c.run(
-        f"cp {TOP_DIR.joinpath('openapi/local_index_openapi.json')} {TOP_DIR.joinpath('openapi/index_openapi.json')}"
-    )
-
-
-@task
+@task(help={"version": "OPTIMADE API version to set"})
```

Comment:
Oohh, can this be added to other tasks as well? If not in this PR, then we should just create an issue for it :)